unity-renderer/Assets/Rendering/Shaders/Toon/Compiled/DepthNormalsOnlyPass.hlsl

```diff
@@ -9,6 +9,7 @@ PackedVaryings vert(Attributes input)
     ApplyGPUSkinning(input, input.tangentOS, input.uv1);
     #endif
     output = BuildVaryings(input);
+    output.normalWS = input.normalOS;
```

Comment:
If we cannot (easily) set the shadergraph to expect a OS normal I would add a comment there for the future. It will be confusing to get OS normals in a variable called `normalWS`.

Reply:
The thing is they are not OS normals, they are actually WS normals. Going to see what I can do about this.

Reply:
Done, added a TODO comment but I should refactor this in the future.